### ISSUE TITLE
.gitignore: ripgrep: error parsing glob

### ISSUE THREAD

**Issue by jzinn** [resque/resque-loner#52]
Ripgrep logs the following:

```
vendor/ruby/2.5.0/gems/resque-loner-1.3.0/.gitignore: line 2: error parsing glob '**.DS_Store': invalid use of **; must be one path component
vendor/ruby/2.5.0/gems/resque-loner-1.3.0/.gitignore: line 3: error parsing glob '**.swp': invalid use of **; must be one path component
```

**Comment by iloveitaly** [comment]
@jzinn can you submit a PR to fix?

**Comment by jzinn** [comment]
It looks like the gitignore syntax rules changed between git 2.19.0 and 2.20.0.

git 2.19.0:
> Other consecutive asterisks are considered invalid.

git 2.20.0:
> Other consecutive asterisks are considered regular asterisks and will match according to the previous rules.

### A fix for users of git 2.19.0:

The fix is just to change `**.DS_Store` to `*.DS_Store` and `**.swp` to `*.swp`.

Ripgrep no longer complains and this project hasn't had a commit in several years, so I won't submit a PR, but here is the diff:

```diff
diff --git a/.gitignore b/.gitignore
index 1018889..132a008 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,6 @@
 *.gem
-**.DS_Store
-**.swp
+*.DS_Store
+*.swp
 Gemfile.lock
 .rvmrc
 spec/stdout
```
